scripts/backfill_downloads.py

```
@@ -85,21 +98,22 @@ def github_request(url):
         if e.code == 403:
             reset = int(e.headers.get("X-RateLimit-Reset", "0"))
             wait = max(reset - int(time.time()), 10)
-            print(f"  Rate limited, sleeping {wait}s...")
+            print(f"  [worker {threading.current_thread().name}] 403 rate-limited, "
+                  f"sleeping {wait}s...", flush=True)
             time.sleep(wait + 1)
-            return github_request(url)
+            return github_request(url, token)
         return None, None
     except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)